roles/core/nic/templates/ifcfg.j2

```diff
@@ -89,3 +89,9 @@ MASTER={{ item.master }}
 {% if item.zone is defined and item.zone is not none %}
 ZONE={{ item.zone }}
 {% endif %}
+
+{# 9. Check if IPv6 has to be disabled #}
+{% if ipv6_disable is defined %}
```

Comment:
This test will match `ipv6_disable: false` and disable ipv6 for this interface. I would use `{% if ipv6_disable | default(false) | bool %}` instead.
Moreover, I believe this should be `item.ipv6_disable`.

Reply:
Dear @MarbolanGos

Any update on this PR ?